media/js/firefox/new/berlin-scene1.js

```diff
@@ -5,11 +5,6 @@
 (function($) {
     'use strict';
 
-    $('.download-link').each(function(i, link) {
-        if (link.href.indexOf('download/thanks/') > -1) {
-            // specify template for scene 2
-            link.href = link.href + '?xv=reggiewatts';
-        }
-    });
-
+    // use custom adjust link for Android/iOS
+    $('.os_android .download-link, .os_ios .download-link').attr('href', 'https://mzl.la/2GEZ3uu');
```

Comment:
I know we likely have no choice here, but to see my phone go from `mzl.la` -> `bit.ly` > `app.adjust` is a pretty painful UX (esp when our download buttons already point people to the right place before this change).

I'd love to know why we really need to do this (do people know our buttons already get people to the right place?). Afaik, we can already track download clicks on mobile in GA, right?